src/main/java/com/google/api/codegen/DiscoveryContext.java

```diff
@@ -27,15 +26,15 @@
 import javax.annotation.Nullable;
 
 /**
- * A LanguageContext that provides helpers specific to the Discovery use case.
+ * A CodegenContext that provides helpers specific to the Discovery use case.
  */
-public abstract class DiscoveryContext extends LanguageContext {
+public abstract class DiscoveryContext extends CodegenContext {
 
   private final Service service;
   private final ApiaryConfig apiaryConfig;
 
   /**
-   * Constructs the abstract instance of the language provider.
+   * Constructs an abstract instance.
```

Comment:
This is rather terse and unhelpful, although the previous version wasn't much better.